jruby-gradle-base-plugin/src/integTest/groovy/com/github/jrubygradle/JRubyExecIntegrationSpec.groovy

```diff
@@ -83,14 +83,18 @@ class JRubyExecIntegrationSpec extends Specification {
 
     def "Running a script that requires a gem, a separate jRuby and a separate configuration"() {
         given:
+            final String newVersion = '1.7.11'
+            assert project.jruby.execVersion != newVersion
```

Comment:
the assert I copied from another test which just makes sure the current default version is different from the newVersion

but the test will fail in this case - so I better omit it for "clarity"